include/eve/algo/algo.hpp

```diff
@@ -0,0 +1,49 @@
+//==================================================================================================
```

Comment:
Why would we have the second copy of the file I don't believe we whould have in the first place?